Chemistry/src/DataModel/OrbitalIntegral/OrbitalIntegral.cs

```diff
@@ -86,11 +85,26 @@ public OrbitalIntegral(IEnumerable<Int64> orbitalIndices, Double coefficient, Co
             }
         }
 
+        public TermType.OrbitalIntegral GetTermType()
+        {
+            switch (Length())
+            {
+                case 0:
+                    return TermType.OrbitalIntegral.Identity;
+                case 2:
+                    return TermType.OrbitalIntegral.OneBody;
+                case 4:
+                    return TermType.OrbitalIntegral.TwoBody;
+                default:
+                    throw new ArgumentException("Attempted to classify unimplemented orbital integral.");
+            }
+        }
+
         /// <summary>
         /// Returns length of <see cref="OrbitalIndices"/>. 
         /// </summary>
         /// <returns>Length of <see cref="OrbitalIndices"/>.</returns>
-        public Int64 Length()
+        public int Length()
```

Comment:
For most collections, `Length` is a property and not a method.
```suggestion
        public int Length => OrbitalIndices.Length
```

Reply:
Fixed